styles/components/Internship/CommitteeInternshipInfo.module.scss

```diff
@@ -0,0 +1,85 @@
+@import '../../variables.scss';
+
+.committee-internship-info {
+  background-color: #FFF;
+  border-radius: 1rem;
+  box-shadow: 0 0.125rem 0.5rem rgba(0, 0, 0, 0.3), 0 0.0625rem 0.125rem rgba(0, 0, 0, 0.2);
+  font-size: 1.25em;
+  margin: auto;
+  margin-bottom: 50px;
+  padding: 1.125em 1.5em;
+  position: relative;
+  width: 80vw;
+  
+  .internship-info-data {
+    font-size: 0.8em;
+    line-height: 150%;
+    margin: 0 22px 14px;
+  }
+
+  &.studio {
+    box-shadow: 0 0.125rem 0.5rem $studio-raspberry, 0 0.0625rem 0.125rem $studio-raspberry;
+  }  
+  
+  &.icpc {
+    box-shadow: 0 0.125rem 0.5rem $icpc-tangerine, 0 0.0625rem 0.125rem $icpc-tangerine;
+  }
+  
+  &.design {
+    box-shadow: 0 0.125rem 0.5rem $design-orange, 0 0.0625rem 0.125rem $design-orange;
+  }
+  
+  &.cyber {
+    box-shadow: 0 0.125rem 0.5rem $cyber-amber, 0 0.0625rem 0.125rem $cyber-amber;
+  }
+  
+  &.teachla {
+    box-shadow: 0 0.125rem 0.5rem $teachla-green, 0 0.0625rem 0.125rem $teachla-green;
+  }
+  
+  &.w {
+    box-shadow: 0 0.125rem 0.5rem $w-teal, 0 0.0625rem 0.125rem $w-teal;
+  }
+  
+  &.ai {
+    box-shadow: 0 0.125rem 0.5rem $ai-arctic, 0 0.0625rem 0.125rem $ai-arctic;
+  }
+  
+  &.hack {
+    box-shadow: 0 0.125rem 0.5rem $hack-purple, 0 0.0625rem 0.125rem $hack-purple;
+  }
+  
+  &.board {
+    box-shadow: 0 0.125rem 0.5rem $acm-cobalt, 0 0.0625rem 0.125rem $acm-cobalt;
+  }
```

Comment:
Would like to generate this with a loop if we can.

Reply:
already said this - but im not sure how to do this since the loop isn't constant due to the color names being different for each committee

Reply:
So you can create a map between committee names and colours; Vincent did this in his PR, here's an example: https://github.com/VeeGee-Creations/website/blob/Reimplement-Navbar/styles/components/Navbar.scss#L130

TL;DR: use `@each` and SASS lists/maps.